lib/smart_app_launch/token_response_body_test.rb

```diff
@@ -10,7 +10,8 @@ class TokenResponseBodyTest < Inferno::Test
       an access token or a message indicating that the authorization request
       has been denied. `access_token`, `token_type`, and `scope` are required.
       `token_type` must be Bearer. `expires_in` is required for token
-      refreshes.
+      refreshes. Unless the `ignore_missing_scopes_check` flag is set in the
+      configuration options, all scopes requested should be approved.
```

Comment:
I'm not sure we should have this language in here like this because the test user can't be expected to locate the source and inspect code to determine the behavior of the test.

If we can't change hte language in the description based on if the flag is set or not, I'd vote to leave the language out.  I know that means we endsus up in the situation where we have a undocumented requirement, which is partly why I was advocating for simply not having this warning logic be put in here at all, but its not a huge deal and we just need this done for the release.